webservices/rest.py

```diff
@@ -73,7 +73,7 @@ def sqla_conn_string():
 
 
 app = Flask(__name__)
-app.debug = True
+# app.debug = True
```

Comment:
You might want to have a way to enable this during development, but this works for now.